lib/internal/webstreams/readablestream.js

```diff
@@ -371,10 +371,7 @@ class ReadableStream {
     // The web platform tests require that these be handled one at a
     // time and in a specific order. options can be null or undefined.
     validateObject(options, 'options', kValidateObjectAllowObjectsAndNull);
-    const preventAbort = options?.preventAbort;
-    const preventCancel = options?.preventCancel;
-    const preventClose = options?.preventClose;
-    const signal = options?.signal;
+    const { preventAbort, preventCancel, preventClose, signal } = options;
```

Comment:
Can't `options` also be `null`?

Reply:
when I looked at the places where it was crossed, I saw that it was protected.

Reply:
The line right above this allows a null value. We should account for that.

What if the user calls this function:
pipeThrough(..., null)

Than this'll throw a destructoring error